src/Movies.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import './Movies.css';
+import Card from './Card';
+
+function Movies({ movies, setSelection, displayMovie }) {
+    const movieCards = 
+    movies.movies.map(movie => {
+        return (
```

Comment:
You could consider destructuring `movie` here so you don't have to keep writing `movie.` over and over. Totally up to you though.